content/100-getting-started/02-setup-prisma/100-start-from-scratch/110-relational-databases/100-connect-your-database.mdx

```diff
@@ -163,6 +188,32 @@ DATABASE_URL="mysql://root:randompassword@localhost:3306/mydb"
 
 </SwitchTech>
 
+<SwitchTech technologies={['node', 'mssql']}>
+
+<NavigationLinksContainer>
+
+<ButtonLink
+  color="dark"
+  type="primary"
+  href="/getting-started/setup-prisma/start-from-scratch/relational-databases-node-mssql"
+  arrowRight
+>
+  Installation
+</ButtonLink>
+
+<ButtonLink
+  color="dark"
+  type="primary"
+  href="./introspect-your-database-node-mssql"
+  arrow
+>
+  Introspect your database
+</ButtonLink>
```

Comment:
broken

Reply:
Ah, good catch, thats from the old introspect link!